lambdas/backend/__tests__/admin-sdk-generation-test.js

```diff
@@ -1,5 +1,5 @@
-const putAdminCatalogSdkGeneration = require('../express-route-handlers').putAdminCatalogSdkGeneration
-const deleteAdminCatalogSdkGeneration = require('../express-route-handlers').deleteAdminCatalogSdkGeneration
+// const putAdminCatalogSdkGeneration = require('../express-route-handlers').putAdminCatalogSdkGeneration
```

Comment:
Do we need to keep these imports?

Reply:
Likely not, and these in particular are broken anyways. We can revisit them later, once I have the chance. (This isn't the only commented-out import, BTW.)